Header header
bool bumper_front
uint8 menu_controls
bool motion_detector
int32 odom_ticksLeft
int32 odom_ticksRight
int32 parkSens_inner
int32 parkSens_outer
uint32 odom_millisSinceLastUpdate
float32 ultrasound_Back
uint8 arm_enabled
int32 arm_yaw
int32 arm_shoulder
int32 arm_elbow
int32 arm_roll
int32 arm_clamp
bool arm_fore_clamp
bool arm_back_clamp
float32 imu_compass_v3_x
float32 imu_compass_v3_y
float32 imu_compass_v3_z
float32 imu_angular_velocity_v3_x
float32 imu_angular_velocity_v3_y
float32 imu_angular_velocity_v3_z
float32 imu_linear_acceleration_v3_x
float32 imu_linear_acceleration_v3_y
float32 imu_linear_acceleration_v3_z
bool imu_gyro_bias_valid
float32 lidar_speed
uint8[] lidar_data
int32[] status_code
int32[] status_param
string log_message
int32 log_count
